.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,37 @@
+<!--- Provide a general summary of the issue in the Title above, if the title is not expressive enough. -->
```

Comment:
Conceptual question: Who are these comments aimed at? They somewhat read like the user creating an issue should be able to read them, but looking at the rich diff which shows the rendered markdown, I cannot see them.

Reply:
> Who are these comments aimed at?

The are aimed for the creator of the issue.

> They somewhat read like the user creating an issue should be able to read them, but looking at the rich diff which shows the rendered markdown, I cannot see them.

The comments are meant to only be seen when creating/editing the issue. The information in the comments only exists to give hints on what to write at the position of the comment. 
Making the text in the comments visible as default would only overcrowd the issue with unnecessary information. (At least in my opinion)

Reply:
Additionally, those comments kinda guide a non-scalable member through the template to try to ensure they understand what information we need / is helpful. I know this does not happen often, but I did happen in the past :)

Reply:
> The comments are meant to only be seen when creating/editing the issue. The information in the comments only exists to give hints on what to write at the position of the comment.

Ah I see, the comments are only visible in non-markdown mode, so when creating/editing the issue. That makes sense :)